src/libraries/restore/runtime/runtime.depproj

```diff
@@ -85,6 +85,27 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="GetCrossgen2"
+          DependsOnTargets="ResolveCoreCLRFilesFromLocalBuild"
+          AfterTargets="AfterResolveReferences;FilterNugetPackages"
+          Condition="'$(RuntimeFlavor)' != 'Mono' and '$(Crossgen2Host)' == 'true'">
+    <ItemGroup>
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/crossgen2.dll" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/crossgen2.deps.json" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/crossgen2.runtimeconfig.json" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/Microsoft.DiaSymReader.dll" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/System.CommandLine.dll" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/ILCompiler.DependencyAnalysisFramework.dll" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/ILCompiler.ReadyToRun.dll " />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/ILCompiler.TypeSystem.ReadyToRun.dll " />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/api-ms-win-core-console-l1-2-0.dll" Condition="'$(TargetOS)' == 'Windows_NT'" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/*mscordaccore_*.*" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/*jitinterface.*" />
+      <CoreCLRFiles Include="$(CoreCLRArtifactsPath)/crossgen2/*clrjitilc.*" />
+      <ReferenceCopyLocalPaths Include="@(CoreCLRFiles)" />
```

Comment:
Why copy this?  Does it need to be in the test shared framework?  I would prefer to move in the direction of making our test framework closer to the real thing rather than dump more test specific changes to it.